### ISSUE TITLE
Create nested elements with FileStorage in Python

### ISSUE THREAD

**Issue by PatrickNa** [opencv/opencv#18117]
##### System information (version)

- OpenCV => 4.1.1
- Operating System / Platform => Ubuntu 18.04
- Installed with anaconda.
- Python 3.6

##### Detailed description

I would like to save camera parameters with with `FileStorage`. The file should also contain nested elements like `resolution{width, height}`. However the creation of the nested elements is not working properly: `startWriteStruct` is not available in Python and` write('elementName', '{')` creates an element that embraces the brackets `<resolution>{</resolution>`

At this point I can not update to the latest version 4.4. Is it still possible to fix it?

##### Steps to reproduce

```python
import cv2

camera_parameters = cv2.FileStorage('./configs/camera_parameters.xml', cv2.FILE_STORAGE_WRITE)
camera_parameters.write('resolution', '{')
camera_parameters.write('width', 1280)
camera_parameters.write('height', 720)
camera_parameters.write('resolution', '}')


camera_parameters.release()
```

##### Issue submission checklist

 - [ ] I report the issue, it's not a question
   
 - [x] I checked the problem with documentation, FAQ, open issues,
       answers.opencv.org, Stack Overflow, etc and have not found solution
   
 - [ ] I updated to latest OpenCV version and the issue is still there
   
 - [ ] There is reproducer code and related data files: videos, images, onnx, etc

**Comment by asmorkalov** [comment]
`startWriteStruct` function is exported to Python tool. Please take a look at [tutorial](https://docs.opencv.org/master/dd/d74/tutorial_file_input_output_with_xml_yml.html): `python/tutorial_code/core/file_input_output/file_input_output.py` and test code: `modules/python/test/test_filestorage_io.py`

**Comment by PatrickNa** [comment]
Yes, I noticed it is available in the latest version, but in 4.1 it seems not to be there yet.

**Comment by asmorkalov** [comment]
OpenCV team does not support old releases. Please update to the latest version. It's 4.4.x for your case.
